src/dft/backends/cufft/commit.cpp

```diff
@@ -315,6 +332,17 @@ class cufft_commit final : public dft::detail::commit_impl<prec, dom> {
         clean_plans();
     }
 
+    static void apply_external_workspace_setting(cufftHandle& handle,
+                                                 const config_value& workspace_setting) {
```

Comment:
```suggestion
    static void apply_external_workspace_setting(cufftHandle handle,
                                                 config_value workspace_setting) {
```
cufftHandle is just an `unsigned int` so should be passed by value, and `config_value` is something like that too.

Reply:
Done in [1da13ee](https://github.com/oneapi-src/oneMKL/pull/411/commits/1da13eef0fbb663fcadc1abbcc15ad930f977589).